docs/usecases/remodnav.rst

```diff
@@ -291,45 +290,137 @@ to fill tables with unique names with minimal work, for example like this:
 
 
-Running the python script hence will print plenty of LaTeX commands to your screen (try it out,
-if you want!). Those statements just need to be captured, and bound to the ``.tex`` file of your
-script.
+Running the python script hence will print plenty of LaTeX commands to your screen (try it out
+in the actual manuscript, if you want!). This was step number 1 of 4.
+
+.. findoutmore:: How about figures?
+
+   To include figures, the figures just need to be saved into a dedicated location (for example
+   a directory ``img/``) and included into the ``.tex`` file with standard ``LaTeX`` syntax.
+   Larger figures with subfigures can be created by combining several figures:
+
+   .. code-block:: tex
+
+      \begin{figure*}[tbp]
+        \includegraphics[trim=0 8mm 3mm 0,clip,width=.5\textwidth]{img/mainseq_lab}
+        \includegraphics[trim=8mm 8mm 0 0,clip,width=.5\textwidth-3.3mm]{img/mainseq_sub_lab} \\
+        \includegraphics[trim=0 0 3mm 0,clip,width=.5\textwidth]{img/mainseq_mri}
+        \includegraphics[trim=8mm 0 0 0,clip,width=.5\textwidth-3.3mm]{img/mainseq_sub_mri}
+
+        \caption{Main sequence of eye movement events during one 15 minute sequence of
+        the movie (segment 2) for lab (top), and MRI participants (bottom). Data
+        across all participants per dataset is shown on the left, and data for a single
+        exemplary participant on the right.}
+
+        \label{fig:overallComp}
+      \end{figure*}
+
+   This figure looks like this in the manuscript:
 
+   .. figure:: ../img/remodnav2.png
+
+For step 2 and 3, the print statements need to be captured and bound to the ``.tex`` file.
 The `tee <https://en.wikipedia.org/wiki/Tee_(command)>`_ command can write all of the output to
 a file (called ``results_def.tex``):
 
 .. code-block:: python
 
    code/mk_figuresnstats.py -s | tee results_def.tex
 
-One can include this file as an input source into the ``.tex`` file with
+This will redirect every print statement the script wrote to the terminal into a file called
+``results_def.tex``. This file will hence be full of ``\newcommand`` definitions that contain
+the results of the computations.
+
+For step 3, one can include this file as an input source into the ``.tex`` file with
 
 .. code-block:: tex
 
    \begin{document}
    \input{results_def.tex}
 
-Upon compilation of the ``.tex`` file into a PDF, the results of the computations captured with
-``\newcommand`` definitions are inserted into the respective part of the manuscript.
+Upon compilation of the ``.tex`` file into a PDF, the results of the
+computations captured with ``\newcommand`` definitions are inserted into the respective part
+of the manuscript.
 
+The last step is to automate this procedure. So far, the script would need to be executed
+with a command line call, and the PDF compilation would require another commandline call.
 To automate this process, `Makefiles <https://en.wikipedia.org/wiki/Make_(software)>`_ can help.
```

Comment:
Maybe state that `make` is a dinosaur from decades ago that is known to many, and its advantage is that it will deliver results regardless of what actually needs to be done. It is always just `make`, regardless of programming language, operating systems, etc.

However, the makefile below will look alien to people who haven't worked with this yet. Would be could to state that any kind of "top-level" script that can be executed without parameter would do the job (in some sense).